.gitignore

```diff
@@ -1,8 +1,15 @@
 # Visual Studio
 .vs/
+.DS_Store
+Notify.userprefs
+NotifyTests/.DS_Store
+
 # NuGet Packages
 *.nupkg
 # The packages folder can be ignored because of Package Restore
 **/packages/*
 bin/
-obj/
\ No newline at end of file
+obj/
+
+# Dev Environment
```

Comment:
Should probably indicate that this line ignores mac dev files, equivalent in windows would probably be a `.bat` or `.ps`?